.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,14 +2,13 @@
 
 <!---
 
-Thank you for contributing to The Foreman, please read the following guide
-available at https://www.theforeman.org/contribute.html - in short:
+Thank you for contributing to The Foreman, please read the [following guide](https://www.theforeman.org/contribute.html)
+- in short:
 
-* Create an issue at https://projects.theforeman.org/projects/foreman/issues
+* Create an issue in [Redmine](https://projects.theforeman.org/projects/foreman/issues)
```

Comment:
This could be:

    [Create an issue](https://projects.theforeman.org/projects/foreman/issues)